render_blender.py

```diff
@@ -26,149 +30,209 @@
                     help='Number of bit per channel used for output. Either 8 or 16.')
 parser.add_argument('--format', type=str, default='PNG',
                     help='Format of files generated. Either PNG or OPEN_EXR')
+parser.add_argument('--resolution', type=int, default=600,
+                    help='Resolution of the images.')
 
 argv = sys.argv[sys.argv.index("--") + 1:]
 args = parser.parse_args(argv)
 
-import bpy
+# Set up rendering
+context = bpy.context
+scene = bpy.context.scene
+render = bpy.context.scene.render
+
+render.engine = 'CYCLES' # ('CYCLES', 'BLENDER_EEVEE', ...)
```

Comment:
Why not leave the default here, i.e. delete the line?

Reply:
Some people may want to use Eevee

Reply:
I agree, the default from blender is even Eevee. I would like the script to stay consistent with that default.

Would your prefer having this exposed as a command line argument?

Reply:
I think it's best to leave this here to help indicate to developers who do not have much experience with BPY (such as myself) to change certain hyper parameter, even if we're not going to make them cmd-line-argument configuratble.

Setting variables and their examples of their possible values, even if it's defaulted correctly, is very helpful for open sourcing.